### PR TITLE
New contraband supply item: 50 butterflies

### PR DESCRIPTION
**Pull request by Incoming5643**
untitled-1 copy
(◕‿◕✿)

### CODE REVIEW

**Comment by TZK13** [comment]
Yes.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -786,6 +786,13 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	containertype = /obj/structure/closet/critter/fox
 	containername = "fox crate"
 
+/datum/supply_packs/organic/butterfly
+	name = "Butterflies Crate"
+	cost = 50.
```
Comment (Iamgoofball) [inline code/datums/supplypacks.dm]: >.

File: code/game/objects/structures/crates_lockers/closets/crittercrate.dm [hunk 2]
```diff
@@ -34,6 +34,9 @@
 			if(prob(50))
 				content_mob = /mob/living/simple_animal/cat/Proc
 			new content_mob(loc)
+		else if(content_mob == /mob/living/simple_animal/butterfly)
```
Comment (Iamgoofball) [inline code/game/objects/structures/crates_lockers/closets/crittercrate.dm]: >not making this a "spawn mob amount" var

Comment (Anonus) [inline code/game/objects/structures/crates_lockers/closets/crittercrate.dm]: at this point you could probably just make the crate contain the animals anyway, as I think the list for unsendable objects is only for station->centcom and not the other way around.

**Comment by ike709** [comment]
@Aranclanos look at that glorious, very informative commit

**Comment by AlexCrimson** [comment]
As someone who prefers SUPER SERIOUS Cargo, and hates all this joke bullshit that makes the department waste its points on silly shit... pls no.

**Comment by TZK13** [comment]
>SUPER SERIOUS Cargo

More like super boring Cargo. :sunglasses:

**Comment by Incoming5643** [comment]
There are tactical reasons to order 50 BUTTERFLIES:

1. It makes moving through an area in harm intent nearly impossible
2. Its an amazing distraction against hostile simple animals
3. it's an amazing meat shield against incoming fire
4. it's 50 butterflies

**Comment by Dorsisdwarf** [comment]
Tactical butterfly bomb as the QM to screw the ops Hail Cargonia/10

**Comment by Kraseo** [comment]
>SUPER SERIOUS Cargo

yeah because having a crate that gives you random hats at the expense of 200 points is super serious

**Comment by AlexCrimson** [comment]
I am well aware of that crate and do not like it.

**Comment by Miauw62** [comment]
No fun allowed.
Op 30-jan.-2015 20:51 schreef "AlexCrimson" <notifications@github.com>:

**Comment by AlexCrimson** [comment]
Hey im all for fun. Its just Cargo has a horrible habit of wasting points and not doing their job. They would rather do their Cargonia shit.

**Comment by Dorsisdwarf** [comment]
The thing is, though - Other than rainbowshirts ordering everything under the sun, the only orders you're really going to reliably take are "MINERALS PLZ CARGO" and "Virus crate plz", and then you're done. Unless of course, it's cult/rev and sec has their groove on.

There's a reason Cargo gets bored and makes autismo forts so often.

**Comment by Incoming5643** [comment]
what is life without temptation?

**Comment by Jalleo** [comment]
@AlexCrimson if you work out something that could have cargo be useful normally while not making a crutches another department go ahead.

**Pull request merged**